src/McpContext.ts

```diff
@@ -131,6 +131,16 @@ export class McpContext implements Context {
     this.#consoleCollector.addPage(page);
     return page;
   }
+  async closePage(pageIdx: number): Promise<void> {
+    if (this.#pages.length === 1) {
+      throw new Error(
```

Comment:
Should we throw an error if this is OK behavior?

Reply:
I think it is fine, the model will get error message explaining the situation.